apis/r/R/SOMADataFrame.R

```diff
@@ -20,12 +20,21 @@ SOMADataFrame <- R6::R6Class(
     #' @param index_column_names A vector of column names to use as user-defined
     #' index columns.  All named columns must exist in the schema, and at least
     #' one index column name is required.
+    #' @param domain An optional list of 2-element vectors specifying the domain of each index
+    #' column. Each vector should be a pair consisting of the minimum and maximum values storable in
+    #' the index column. For example, if there is a single int64-valued index column, then `domain`
+    #' might be `c(100, 200)` to indicate that values between 100 and 200, inclusive, can be stored
+    #' in that column.  If provided, this list must have the same length as `index_column_names`,
+    #' and the index-column domain will be as specified.  If omitted entirely, or if `NA` in a given
```

Comment:
The checks were for `NULL`, not `NA`
```suggestion
    #' and the index-column domain will be as specified.  If omitted entirely, or if `NULL` in a given
```